ec2/TemplateClusterCreate-NoVPC.json

```diff
@@ -585,7 +603,7 @@
                 "memTotalMb=$(($memTotalKb / 1024)) \n ",
                 "tmp=$(($memTotalMb * 90))  \n",
                 "xmxMb=$(($tmp / 100))  \n",
-                "sudo nohup java -Xmx${xmxMb}m -jar /opt/h2oai/h2o.jar -flatfile /opt/h2oai/flatfile.txt 1> /dev/null 2> /opt/h2oai/h2o.err & \n",
+                "sudo nohup java -Xmx${xmxMb}m -jar /opt/h2oai/h2o.jar -hash_login -login_conf /opt/h2oai/realm.properties -flatfile /opt/h2oai/flatfile.txt 1> /dev/null 2> /opt/h2oai/h2o.err & \n",
```

Comment:
I assume we will have to modify/document script that populates the realm.properties.